src/main/java/com/czertainly/core/service/cmp/util/NullUtil.java

```diff
@@ -0,0 +1,31 @@
+package com.czertainly.core.service.cmp.util;
+
+import java.util.function.Supplier;
+
+public class NullUtil {

```

Comment:
Is this needed? Can we move it package `com.czertainly.core.util`?

Reply:
DONE, moved into util package